cmd/homi/docker/service/validator.go

```diff
@@ -123,51 +125,49 @@ var validatorTemplate = `{{ .Name }}:
+        echo 'NO_DISCOVER=1' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
+        echo 'SYNCMODE="full"' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
+        echo 'ADDITIONAL="$$ADDITIONAL --identity \"{{ .Name }}\""' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
+        echo 'ADDITIONAL="$$ADDITIONAL --nodekeyhex {{ .NodeKey }}"' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
 {{- if .AddPrivKey}}
-        --unlock "75a59b94889a05c03c66c3c84e9d2f8308ca4abd" \
-        --password "/klaytn/password.txt" \
+        echo 'ADDITIONAL="$$ADDITIONAL --unlock \"75a59b94889a05c03c66c3c84e9d2f8308ca4abd\""' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
+        echo 'ADDITIONAL="$$ADDITIONAL --password "/klaytn/password.txt"' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
 {{- end}}
 {{- if eq .NodeType "scn" }}
-        --scconsensus "istanbul" \
+        echo 'PORT=32323' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
+        echo 'ADDITIONAL="$$ADDITIONAL --scconsensus "istanbul""' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
 {{- else if eq .NodeType "spn" }}
-        --scconsensus "istanbul" \
+        echo 'ADDITIONAL="$$ADDITIONAL --scconsensus "istanbul""' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
 {{- else if eq .NodeType "sen" }}
-        --scconsensus "istanbul" \
+        echo 'ADDITIONAL="$$ADDITIONAL --scconsensus "istanbul""' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
 {{- end}}
 {{- if .ParentChainId }}
-        --parentchainid {{ .ParentChainId }} \
-        --subbridge \
-        --subbridgeport 50506 \
+        echo 'ADDITIONAL="$$ADDITIONAL --parentchainid {{ .ParentChainId }}"' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
+        echo 'SC_SUB_BRIDGE=1' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
+        echo 'SC_SUB_BRIDGE_PORT=50506' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
 {{- end}}
 {{- if .UseFastHttp}}
-        --srvtype fasthttp \
+        echo 'SERVER_TYPE=fasthttp' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
 {{- end}}
 {{- if eq .NodeType "cn" }}
-        --rewardbase {{ .Address }}
+        echo 'REWARDBASE={{ .Address }}' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
 {{- else if eq .NodeType "pn" }}
-        --txpool.nolocals
+        echo 'ADDITIONAL="$$ADDITIONAL --txpool.nolocals"' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
 {{- else if eq .Name "EN-0" }}
-        --mainbridge \
-        --mainbridgeport 50505
+        echo 'SC_MAIN_BRIDGE=1' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
+        echo 'SC_MAIN_BRIDGE_PORT=50505' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
+        echo 'SC_MAIN_BRIDGE_INDEXING=1' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
 {{- end }}
-
+        echo 'ADDITIONAL="$$ADDITIONAL --debug --metrics --prometheus"' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
+        /klaytn-docker-pkg/bin/k{{ .NodeType }}d start
+        while true; do sleep 1; done
```

Comment:
Trick is to keep containers running even node process is down.  Debugging purpose.